test/e2e/src/liveExecutionTracker.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {browser} from 'protractor';
+
+import {DiagramWithCallActivityAndTasks} from './diagrams/diagramWithCallActivityAndTasks';
+import {DiagramWithUserTask} from './diagrams/diagramWithUserTask';
+import {DiagramDetail} from './pages/diagramDetail';
+import {DiffView} from './pages/diffView';
+import {LiveExecutionTracker} from './pages/liveExecutionTracker';
+import {RouterView} from './pages/routerView';
+import {StatusBar} from './pages/statusBar';
+
+describe('Diff view', () => {
+
+  let routerView: RouterView;
+  let targetDiagram: DiagramWithUserTask;
+  let diagram: DiagramWithCallActivityAndTasks;
+  let liveExecutionTracker: LiveExecutionTracker;
+
+  const applicationUrl: string = browser.params.aureliaUrl;
+
+  beforeAll(async() => {
+    routerView = new RouterView();
+    targetDiagram = new DiagramWithUserTask();
+    diagram = new DiagramWithCallActivityAndTasks(targetDiagram.name);
+
+    await targetDiagram.deployDiagram();
+    await diagram.deployDiagram();
+    await diagram.startProcess();
+    liveExecutionTracker = new LiveExecutionTracker(applicationUrl, diagram.correlationId, diagram.name, diagram.processInstanceId);
+  });
+
+  afterAll(async() => {
+
+    await diagram.deleteDiagram();
+    await targetDiagram.deleteDiagram();
+  });
+
+  beforeEach(async() => {
+    await routerView.show();
+    await liveExecutionTracker.show();
+  });
+
+  it('should display the diagram when on Live Execution Tracker.', async() => {
```

Comment:
```suggestion
  it('should display the diagram.', async() => {
```

Otherwise this sentence would form:
"The LET should display a diagram when on LET."

One time LET is enough I guess.